pkg/crd/api/targetconfig/v1alpha1/doc.go

```diff
@@ -0,0 +1,5 @@
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen=package
+// +groupName=wgpolicyk8s.io
```

Comment:
the CRD should not use `wgpolicyk8s.io` as group. Policy Reporter is not part of the wg policy org.